fulp_modules/features/antagonists/bloodsuckers/code/bloodsucker/bloodsucker_datum.dm

```diff
@@ -54,6 +54,10 @@
 	var/frenzy_threshold = FRENZY_MINIMUM_THRESHOLD_ENTER
 	///If we are currently in a Frenzy
 	var/frenzied = FALSE
+	///If we are currently in Torpor
+	///Set on entering and exiting Torpor, but not practically used much otherwise.
+	///Originally implemented to fix a bug.
+	var/torpored = FALSE
```

Comment:
dude...

Reply:
Torpor honestly could be made a status effect.

Reply:
> Torpor honestly could be made a status effect.

I'll try turning it into one.

Reply:
I have converted Torpor into a status effect. While doing so I noticed that there are a lot of instances in Bloodsucker code where Torpor is checked for with `HAS_TRAIT(owner.current, TRAIT_NODEATH)`. Shall I replace those with `owner.current.has_status_effect(/datum/status_effect/torpor)`?